### DISCUSSION TITLE
【解決済】組立中のkeyball61で縦2キーが反応しない問題の解決策・原因の方法についてアドバイスお願いします。

### DISCUSSION THREAD

**Discussion by miha-398** [Yowkees/keyball#716] [Q&A]
【問題】
1枚目の写真の赤丸で囲った右端の縦
2つのキーが反応しません。
![IMG_3293](https://github.com/user-attachments/assets/543b72b9-52a8-4def-b9b0-780ea3cb4141)

【自身で試したこと】
2枚目の赤丸の部品について、向きの確認、付け直しを実施しました。
また同じ列の場合はProMicroに問題がある可能性があるとのことで、はんだの付け直しも実施しました。
![IMG_3295](https://github.com/user-attachments/assets/8160028f-047c-4c88-a579-4d892677796f)

【アドバイスのお願い】
3枚目の写真は裏目全体の写真となります。
上記の問題や写真を見て、どのように解決すればよいか、また原因箇所を特定する方法について案があれば、ご教示ください。
![IMG_3294](https://github.com/user-attachments/assets/74c4dbd7-7370-439b-a88a-7b362494a52d)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

本件の２キーに関係するのは、下記の枠で囲った３つのダイオードになります。
特に青枠で示したものが半田不良すると、２キーとも動かなくなるので、このダイオードが原因の可能性が濃厚です。
お手数ですが、方向の確認と半田付け不良がないかご確認ください。

![KB61left2key](https://github.com/user-attachments/assets/beac5b6f-5dee-4b97-b4ff-46451fc6f7a9)

以上、お手数ですがご確認お願いいたします。

**Reply by miha-398** [reply]
ご返信ありがとうございます。問題が解消しました。

Yowkeesさんから指摘頂いた青枠のダイオードの向きが間違っており、
正しく半田付けしたところ、問題のあった2つのキーが改善されて反応するようになりました。

誠にありがとうございました。

**Comment by Yowkees** [comment]
無事に問題が解消したようで良かったです！
ぜひ末永くKeyball61をご愛用いただければ幸いです。
